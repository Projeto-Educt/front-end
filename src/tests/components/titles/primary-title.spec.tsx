import PrimaryTitle from '@/components/titles/primary-title';
import { render, screen } from '@testing-library/react';

describe('PrimaryTitle', () => {
  it('should render correctly', () => {
    render(<PrimaryTitle title="Faça seu Registro" />);

    const heading = screen.getByRole('heading', { name: 'Faça seu Registro' });

    expect(heading).toBeInTheDocument();
  });
  it('should match snapshot', () => {
    const { container } = render(<PrimaryTitle title="Faça seu Registro" />);

    expect(container).toMatchSnapshot();
  });
});
