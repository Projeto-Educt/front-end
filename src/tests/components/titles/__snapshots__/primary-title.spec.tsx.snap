// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryTitle should match snapshot 1`] = `
<div>
  <h1
    class="text-3xl font-bold text-primary-600 text-center sm:text-start"
  >
    Fa√ßa seu Registro
  </h1>
</div>
`;
